kani-compiler/src/session.rs

```diff
@@ -61,7 +61,8 @@ static JSON_PANIC_HOOK: LazyLock<Box<dyn Fn(&panic::PanicHookInfo<'_>) + Sync +
                 Lrc::new(SourceMap::new(FilePathMapping::empty())),
                 fallback_bundle,
                 false,
-                HumanReadableErrorType::Default(ColorConfig::Never),
+                HumanReadableErrorType::Default,
```

Comment:
Also add https://github.com/rust-lang/rust/pull/128806

Reply:
Was already added, its item 6 in the list of PR's